### DISCUSSION TITLE
JavaCV Works Locally but Fails on Kubernetes with could not initialize class org.bytedeco.ffmpeg.global.avutil

### DISCUSSION THREAD

**Discussion by CheSauRie** [bytedeco/javacv#2306] [General]
Description

I am using JavaCV with OpenCV and FFmpeg for video processing in my Java application. Everything works perfectly on my local machine, but when I deploy the application to Kubernetes, I encounter the following error:

java.lang.NoClassDefFoundError: Could not initialize class org.bytedeco.ffmpeg.global.avutil

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacpp-presets/issues/1379
